pyproject.toml

```
@@ -3,7 +3,7 @@ requires = ["setuptools >= 30.3.0", "wheel"]
 
 
 [tool.black]
-target-version = ["py37", "py38"]
+target-version = ["py37", "py38", "py39", "py310"]
```

Black's target version is for indicating what *older* releases you still need support for, so `py310` isn't an available choice here yet.

```suggestion
target-version = ["py37", "py38", "py39"]
```

---

According to this issue, you are supposed to list all versions of python that you want to support, not just legacy versions:

https://github.com/psf/black/issues/751

The relevant comment on that issue was made by the person who implemented the feature to begin with:

https://github.com/psf/black/pull/618

So I think listing Python 3.9 and Python 3.10 is appropriate.

---

So let's just go with author's answer in that first linked discussion. And it's always nice to be explicit.

> You should include all Python versions that you want your code to run under

---

Ok, if so then I'm all in. Previously it failed because Black didn't recognize the latest Python version.